dash-renderer/src/persistence.js

```diff
@@ -265,10 +265,22 @@ const noopTransform = {
     apply: (storedValue, _propValue) => storedValue,
 };
 
-const getTransform = (element, propName, propPart) =>
-    propPart
-        ? element.persistenceTransforms[propName][propPart]
-        : noopTransform;
+const getTransform = (element, propName, propPart) => {
+    if (typeof element.persistenceTransforms !== 'undefined') {
```

Comment:
Here you can simplify the top level check to:
```suggestion
    if (element.persistenceTransforms) {
```
any truth-y value will pass the test (https://dorey.github.io/JavaScript-Equality-Table/)

Reply:
91c5324